src/main/java/frc/robot/RobotContainer.java

```diff
@@ -356,44 +359,77 @@ private void configureButtonBindings() {
                 : Commands.runOnce(() -> mapleSimArenaSubsystem.setRobotHasCoral(true)));
     coralOuttakeButton.whileTrue(intake.floorOuttake());
 
-    Logger.recordOutput("Intake/Modes/L1 Score Mode", !elevatorNotL1);
-    Logger.recordOutput("Intake/Modes/Algae Mode", intakeAlgeaMode);
-    Logger.recordOutput("Intake/Modes/Coral Station Mode", coralStationMode);
+    Logger.recordOutput("Intake/Modes/L1 Score Mode", elevatorL1);
+    Logger.recordOutput("Intake/Modes/Algae Mode", intakeMode == IntakeModeEnum.ALGAE);
+    Logger.recordOutput(
+        "Intake/Modes/Coral Station Mode", intakeMode == IntakeModeEnum.CORAL_STATION);
     OI.getButton(OI.Operator.Y)
         .onTrue(
             Commands.runOnce(
                 () -> {
-                  elevatorNotL1 = !elevatorNotL1;
-                  Logger.recordOutput("Intake/Modes/L1 Score Mode", !elevatorNotL1);
+                  elevatorL1 = !elevatorL1;
+                  switch (intakeMode) {
+                    case SCORE_L1:
+                      intakeMode = IntakeModeEnum.GROUND_INTAKE;
```

Comment:
What if its supposed to do coral station mode? 
The states overlap, which is the reason we weren't using an enum in first place.